Sources/DolbyIORTSCore/StreamCoordinator.swift

```diff
@@ -8,6 +8,10 @@ import MillicastSDK
 import os
 
 open class StreamCoordinator {
+    public struct Configuration {
+        let retryOnConnectionError = true
```

Comment:
Just one parameter now. Is it worth having a configuration struct?

Reply:
It's `let`. Will you consider to use`static`?

Reply:
My thinking was that to group related data together in its on configuration object, than adding properties to StreamCoordinator.